components/addressmanager/src/lib.rs

```diff
@@ -359,7 +359,7 @@ mod address_store_with_cache {
 
     impl Store {
         fn new(db: Arc<DB>) -> Self {
-            let db_store = DbAddressesStore::new(db, 0);
+            let db_store = DbAddressesStore::new(db, CachePolicy::Empty);
```

Comment:
Not specifically related to this PR, but the fact that this store's cache is empty, that it's not a test AND it's very specifically a mod that says `address_store_with_cache` makes me think this use of cache here needs to be re-examined (even outside the context of this PR).

Reply:
Explained in a comment